src/main/java/io/appium/java_client/flutter/SupportsFlutterCameraMocking.java

```diff
@@ -0,0 +1,47 @@
+package io.appium.java_client.flutter;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Base64;
+import java.util.Map;
+
+/**
+ * This interface extends {@link CanExecuteFlutterScripts} and provides methods
+ * to support mocking of camera inputs in Flutter applications.
+ */
+public interface SupportsFlutterCameraMocking extends CanExecuteFlutterScripts {
+
+    /**
+     * Injects a mock image into the Flutter application using the provided file.
+     *
+     * @param image the image file to be mocked
+     * @return an {@code Integer} representing the result of the injection operation
+     * @throws IOException if an I/O error occurs while reading the image file
+     */
+    default String injectMockImage(File image) throws IOException {
+        String base64EncodedImage = Base64.getEncoder().encodeToString(Files.readAllBytes(image.toPath()));
+        return injectMockImage(base64EncodedImage);
+    }
+
+    /**
+     * Injects a mock image into the Flutter application using the provided Base64-encoded image string.
+     *
+     * @param base64Image the Base64-encoded string representation of the image
```

Comment:
same as below

Reply:
I have added the file format for now. But size and limit can be updated as per the driver implementation https://github.com/AppiumTestDistribution/appium-flutter-integration-driver/issues/77